src/integration_testing/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+#//////////////////////////////////////////////////////////////////////////////////////
+#// This file is distributed under the University of Illinois/NCSA Open Source License.
+#// See LICENSE file in top directory for details.
+#//
+#// Copyright (c) 2025 QMCPACK developers.
+#//
+#// File developed by: Peter Doak, doakpw@ornl.gov, Oak Ridge National Laboratory
+#//////////////////////////////////////////////////////////////////////////////////////
+
+add_library(integration_testing MockGoldWalkerElements.cpp)
```

Comment:
Could you document what level of integration this CMake target serves?

Reply:
I've tried to clarify this a good bit. I've also change the targets name as a result.